README.md

```diff
@@ -50,3 +50,18 @@ You should see output similar to the following:
  * Debugger PIN: 226-556-590
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+## Module -2 
```

Comment:
We should update these instructions, and ideally the `.env.template` file, to help new developers know what environment variables are expected due to this change

Reply:
This is done as well!